dotnetv3/S3/non-versioned-examples/DeleteObjectExample/DeleteObject.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+
+/// <summary>
+/// This example shows how to delete an object from a non-versioned Amazon
+/// Simple Storage Service (Amazon S3) bucket. This example was created
+/// using the AWS SDK 3.7 for .NET and .NET Core 5.0.
+/// </summary>
+namespace DeleteObjectExample
+{
+    // snippet-start:[S3.dotnetv3.DeleteObjectExample]
+    using System;
+    using System.Threading.Tasks;
+    using Amazon.S3;
+    using Amazon.S3.Model;
+
+    public class DeleteObject
+    {
+        /// <summary>
+        /// The Main method initializes the necessary variables and then calls
+        /// the DeleteObjectNonVersionedBucketAsync method to delete the object
+        /// named by the keyName parameter.
+        /// </summary>
+        public static async Task Main()
+        {
+            const string bucketName = "doc-example-bucket";
+            const string keyName = "testfile.txt";
+
+            // If the Amazon S3 bucket is located in an AWS Region other than the
+            // Region of the default account, specify the bucket's Region in
```

Comment:
specify the bucket's Region -> define the AWS Region for the Amazon S3 bucket

Reply:
done